.vib/osclass/goss/goss.yaml

```diff
@@ -0,0 +1,29 @@
+http:
+  https://osclass:{{ .Vars.service.ports.https }}:
+    status: 200
+    allow-insecure: true
```

Comment:
I have checked the associated test in the description about port `444` and the related error in the [execution-graph](https://cp.bromelia.vmware.com/v1/execution-graphs/548980de-213e-4df5-902a-f31f7a448df8/tasks/b0506ae3-6dd3-4b18-9714-3aee51e26a52/result) is:

```json
{
"duration": 0.003,
"name": "HTTP http://osclass:444 status",
"location": "goss",
"message": "ERRORS: [HTTP: http://osclass:444: status: doesn't match, expect: [200] found: [400]] FAILURES: [HTTP: http://osclass:444: status: doesn't match, expect: [200] found: [400]] ",
"test_id": "4bff62fd",
"status": "FAILED"
},
```

The error is a 400 Bad Request, which does mean that GOSS was able to connect to that port and got a response. Hence, I don't think the actual change of port is failing. The error might be related to accessing a HTTPs port using a plain HTTP protocol. Could you please check it?